### ISSUE TITLE
TypeError on startup on Raspberry Pi Zero W

### ISSUE THREAD

**Issue by ebaauw** [oznu/homebridge-config-ui-x#1251]
**Describe The Bug:**

Homebridge Config UI X throws a TypeError during startup on a Raspberry Pi Zero W.  I haven't seen this on any of my other servers.  Checking the old logs, the error occurred on previous versions of Homebridge Config UI X and NodeJS as well.

Homebridge and Homebridge Config UI X appear to be functioning normally otherwise.

**To Reproduce:**

Restart Homebridge.

**Expected behavior:**

No TypeError.

**Logs:**

```
[Dec 15 21:55:18] [15/12/2021, 21:55:18] [HB Supervisor] Homebridge Storage Path: /var/lib/homebridge
[Dec 15 21:55:18] [15/12/2021, 21:55:18] [HB Supervisor] Homebridge Config Path: /var/lib/homebridge/config.json
[Dec 15 21:55:19] [15/12/2021, 21:55:19] [HB Supervisor] OS: Linux 5.10.63+ arm
[Dec 15 21:55:19] [15/12/2021, 21:55:19] [HB Supervisor] Node.js v16.13.1 /usr/local/bin/node
[Dec 15 21:55:19] [15/12/2021, 21:55:19] [HB Supervisor] Homebridge Path: /usr/local/lib/node_modules/homebridge/bin/homebridge
[Dec 15 21:55:19] [15/12/2021, 21:55:19] [HB Supervisor] UI Path: /usr/local/lib/node_modules/homebridge-config-ui-x/dist/bin/standalone.js
[Dec 15 21:56:09] [15/12/2021, 21:56:09] [Homebridge UI] Homebridge Config UI X v4.41.4 is listening on :: port 8581
[Dec 15 21:56:15] [15/12/2021, 21:56:15] [HB Supervisor] Delaying Homebridge startup by 20 seconds on low powered server
[Dec 15 21:56:18] [15/12/2021, 21:56:18] [Homebridge UI] TypeError: Invalid Version: undefined
[Dec 15 21:56:18] [15/12/2021, 21:56:18] [Homebridge UI] Caught Unhandled Rejection Error :: Details Below
[Dec 15 21:56:18] TypeError: Invalid Version: undefined
[Dec 15 21:56:18]     at new SemVer (/usr/local/lib/node_modules/homebridge-config-ui-x/node_modules/semver/classes/semver.js:19:13)
[Dec 15 21:56:18]     at compare (/usr/local/lib/node_modules/homebridge-config-ui-x/node_modules/semver/functions/compare.js:3:3)
[Dec 15 21:56:18]     at Object.gt (/usr/local/lib/node_modules/homebridge-config-ui-x/node_modules/semver/functions/gt.js:2:29)
[Dec 15 21:56:18]     at StatusService.checkHomebridgeStatus (/usr/local/lib/node_modules/homebridge-config-ui-x/dist/main.js:77153:54)
[Dec 15 21:56:18]     at StatusService.getHomebridgeStats (/usr/local/lib/node_modules/homebridge-config-ui-x/dist/main.js:77149:32)
[Dec 15 21:56:18]     at StatusService.watchStats (/usr/local/lib/node_modules/homebridge-config-ui-x/dist/main.js:77119:53)
[Dec 15 21:56:18]     at StatusGateway.serverStatus (/usr/local/lib/node_modules/homebridge-config-ui-x/dist/main.js:76781:28)
[Dec 15 21:56:18]     at /usr/local/lib/node_modules/homebridge-config-ui-x/node_modules/@nestjs/websockets/context/ws-context-creator.js:43:33
[Dec 15 21:56:18]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[Dec 15 21:56:18]     at StatusGateway.<anonymous> (/usr/local/lib/node_modules/homebridge-config-ui-x/node_modules/@nestjs/websockets/context/ws-proxy.js:12:32)
[Dec 15 21:56:18]     at WebSocketsController.pickResult (/usr/local/lib/node_modules/homebridge-config-ui-x/node_modules/@nestjs/websockets/web-sockets-controller.js:91:24)
[Dec 15 21:56:35] [15/12/2021, 21:56:35] [HB Supervisor] Starting Homebridge with extra flags: -T -D
[Dec 15 21:56:35] [15/12/2021, 21:56:35] [HB Supervisor] Starting Homebridge with custom env: {"NODE_OPTIONS":"--trace-warnings"}
[Dec 15 21:56:35] [15/12/2021, 21:56:35] [HB Supervisor] Started Homebridge v1.3.8 with PID: 22524
[Dec 15 21:56:46] Loaded config.json with 0 accessories and 1 platforms.
[Dec 15 21:56:47] Loaded 0 cached accessories from cachedAccessories.
[Dec 15 21:56:47] ---
[Dec 15 21:57:03] Loaded plugin: homebridge-config-ui-x@4.41.4
[Dec 15 21:57:03] Registering platform 'homebridge-config-ui-x.config'
[Dec 15 21:57:03] ---
[Dec 15 21:57:03] Loading 1 platforms...
[Dec 15 21:57:03] [Config] Initializing config platform...
[Dec 15 21:57:03] [Config] Running in Service Mode
[Dec 15 21:57:03] Publishing bridge accessory (name: pi6, publishInfo: {
[Dec 15 21:57:03]   username: 'CC:22:3D:E3:C7:36',
[Dec 15 21:57:03]   port: 51826,
[Dec 15 21:57:03]   pincode: '***-**-***',
[Dec 15 21:57:03]   category: 2,
[Dec 15 21:57:03]   bind: undefined,
[Dec 15 21:57:03]   mdns: undefined,
[Dec 15 21:57:03]   addIdentifyingMaterial: true,
[Dec 15 21:57:03]   advertiser: 'ciao'
[Dec 15 21:57:03] }).
[Dec 15 21:57:04] Setup Payload:
[Dec 15 21:57:04] X-HM://00253QFZSMSVM
[Dec 15 21:57:04] Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
[Dec 15 21:57:04]                        
[Dec 15 21:57:04]     ┌────────────┐     
[Dec 15 21:57:04]     │ xxx-xx-xxx │     
[Dec 15 21:57:04]     └────────────┘     
[Dec 15 21:57:04]                        
[Dec 15 21:57:04] Homebridge v1.3.8 (HAP v0.9.7) (pi6) is running on port 51826.
```

**Homebridge Config:**
```json
{
    "bridge": {
        "name": "pi6",
        "username": "CC:22:3D:E3:C7:36",
        "port": 51826,
        "pin": "xxx-xx-xxx",
        "advertiser": "ciao"
    },
    "plugins": [
        "homebridge-config-ui-x"
    ],
    "platforms": [
        {
            "platform": "config",
            "name": "Config",
            "port": 8581
        }
    ],
    "accessories": []
}
```

**Screenshots:**

**Environment:**
* **Node.js Version**: v16.13.1
* **NPM Version**: 8.3.0
* **Homebridge Version**: 1.3.8
* **Homebridge Config UI X Version**: v4.41.4 
* **Operating System**: Raspbian GNU/Linux 10 (buster) on a Raspberry Pi Zero W 1.1 (512MB)
* **Process Supervisor**: hb-service

**Comment by oznu** [comment]
Thanks @ebaauw - this is fixed in v4.41.5-test.1

**Comment by ebaauw** [comment]
I can confirm v4.41.5-test.2 no longer throws the error.  Thanks.
